### PR TITLE
Access: Minor Rework

### PR DESCRIPTION
**Pull request by Marukesu**
Some minimal changes to the access classes, the more siginificant one is that we don't set skip_taskbar_hint anymore since that is a X11-only feature and actually could make find the dialog a bit hader if it was open as non-modal and another window take focus.

### CODE REVIEW

**Comment by danirabbit** [comment]
@Marukesu can you resolve conflicts here?

**Comment by Marukesu** [comment]
@danirabbit done

File: src/Access/Dialog.vala [hunk 1]
```diff
@@ -75,27 +73,28 @@ public class Access.Dialog : Granite.MessageDialog {
 
         custom_bin.orientation = Gtk.Orientation.VERTICAL;
         custom_bin.spacing = 6;
+    }
+
+    public override void show () {
```
Comment (danirabbit) [inline src/Access/Dialog.vala:78]: Can you explain why it's better to override `show` than to connect to `realize`? Is there an issue that this fixes?

Comment (Marukesu) [reply src/Access/Dialog.vala:78]: i don't think this should change the current behaviour, this is more a organization/simplification change (separate widget code from windowing code, reduce the number of functions used).

**Review by danirabbit — APPROVED** [review]

**Pull request merged**